src/components/cylc/Info.vue

```diff
@@ -0,0 +1,261 @@
+                :class="{satisfied: prereq.satisfied}"
+              >{{ prereq.expression }}</span>
+              <ul>
+                <li
+                  v-for="condition in prereq.conditions"
+                  :key="condition.taskAlias"
+                >
+                  <span
+                    class="prerequisite-alias condition"
+                    :class="{satisfied: condition.satisfied}"
+                  >
+                    {{ condition.exprAlias }}
+                  </span>
+                  <span style="margin-left: 0.5em">
+                    {{ condition.taskId }}:{{ condition.reqState }}
+                  </span>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </v-expansion-panel-text>
+      </v-expansion-panel>
+
+      <!-- The outputs -->
+      <v-expansion-panel class="outputs-panel">
+        <v-expansion-panel-title color="blue-grey-lighten-1">
+          Outputs
+        </v-expansion-panel-title>
+        <v-expansion-panel-text>
+          <ul>
+            <li v-for="output in outputs" :key="output.label">
+              <span
+                class="condition"
+                :class="{satisfied: output.satisfied}"
+              >{{ output.label }}</span>
+            </li>
+          </ul>
+        </v-expansion-panel-text>
+      </v-expansion-panel>
+
+    </v-expansion-panels>
+  </div>
+</template>
+
+<script>
+import { useJobTheme } from '@/composables/localStorage'
+import GraphNode from '@/components/cylc/GraphNode.vue'
+
+export default {
+  name: 'InfoComponent',
+
+  components: {
+    GraphNode,
+  },
+
+  props: {
+    task: {
+      required: true,
+    },
+    panelExpansion: {
+      required: false,
+      default: [0],
+    },
+  },
+
+  setup (props, { emit }) {
+    return {
+      jobTheme: useJobTheme(),
+    }
+  },
+
+  computed: {
+    panelExpansionModel: {
+      get () {
+        return this.panelExpansion
+      },
+      set (value) {
+        this.$emit('update:panelExpansion', value)
+      },
+    },
+
+    taskMetadata () {
+      // Default task metadata sections (title, description, URL)
+      return this.task?.node?.task?.meta || {}
+    },
+
+    customMetadata () {
+      // Used-defined task metadata sections
+      return this.task?.node?.task?.meta.userDefined || {}
+    },
+
+    prerequisites () {
+      // Task prerequsite information.
+      return this.task?.node?.prerequisites || {}
+    },
+
+    outputs () {
+      // Task output information.
+      return this.task?.node?.outputs || {}
+    },
+
+  },
+
+}
+</script>
+
+<style scoped lang="scss">
+  .c-info {
+    padding: 0.5em;
+
+    // for user-defined text where whitespace should be preserved
+    .markup {
+      white-space: pre;
+    }
+
+    // prefixes a tick or cross before the entry
+    .condition:before {
+      content: '✕';
+      padding-right: 0.5em;
+      color: rgb(255, 100, 100);
+    }
```

Comment:
I think that this makes more sense: Each output can be produced or not, where as to my mind the tick/cross suggests a more succeed/failed outcome.
```suggestion
      content: '☐';
```

```suggestion
    .condition:before {
      content: '☐';
      padding-right: 0.5em;
      color: rgb(0, 0, 0);
    }
```

Reply:
I don't think an empty box is clearer than a cross?

This view copies the format used in the scheduler logs and the `cylc view` command.

Reply:
I agree with Tim's point about green/red being interpreted as success/error.

In the screenshot it looks like the `failed` output is a good thing, while 

![image](https://github.com/user-attachments/assets/3003b857-8dfa-44cc-b88f-049ff5543637)

Also would good to differentiate expected outputs, e.g. by greying out `expired`, `submit-failed` etc. if they are not completed

Reply:
We don't have required / optional information available via GraphQL at the moment, so all we can do is report if an output has been completed or not. We cannot determine whether the completion or non-completion of an output is a good or a bad thing without that information